cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -199,7 +199,7 @@ def render_and_launch_ingress(kube_dns):
         hookenv.open_port(443)
     else:
         hookenv.log('Deleting the http backend and ingress.')
-        kubectl_manifest('delete', '/etc/kubernetes/addons/default-http-backend.yaml')
+        kubectl_manifest('delete', '/etc/kubernetes/addons/default-http-backend.yaml')  # noqa
```

Comment:
We really need to get a linter on PR's so we don't merge these flake8 issues from contributions. :|